Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -2463,6 +2463,8 @@ RenderElement* ComputedStyleExtractor::styledRenderer() const
 {
     if (!m_element)
         return nullptr;
+    if (m_pseudoElementSpecifier == PseudoId::Invalid)
+        return nullptr;
```

Comment:
@nt1m 
I tried to find a way to nullify the element but without success.
I managed a point to have the desired result by touching Element.cpp, but I clearly nullified the style for the parent element itself. 😮‍💨 

So I need a bit of help here. 

Does `m_element` can be modified? or is it constant?

And where could I do something like `m_element = nullptr;`


The parsing is all good. This is just the last stretch where I"m hitting the wall.